### DISCUSSION TITLE
How can I create a route model binding with v6 ?

### DISCUSSION THREAD

**Discussion by franckDev21** [adonisjs/core#4524] [Help] [answered]
I'd like to be able to do something like this please: 

`
Route.get('posts/:post/comments/:comment', 'PostsController.show')
`

**Comment by RomainLanz** [comment, marked as answer]
Hi?

The route model binding package has not been migrated to work on AdonisJS 6 yet.
But nothing stops you from writing exactly the route you shared and handling the data fetching yourself. It is like 3 lines of code.

**Reply by franckDev21** [reply]
Salut Romain !  Merci.
Je vais donc écrire ces 3 lignes de code 😅. C'est vrai que c'est pas beaucoup. Merci, et Adonis JS est VRAIMENT SUPER.
